tests/common.sh

```diff
@@ -275,12 +277,12 @@ function wait_for_backpack() {
 function check_es() {
```

Comment:
Why did this change?

Reply:
You're right, this change is not related with the goal of the PR. In the other hand, it fixes this function, since we call it from CI and we evaluate its return code, this was obviously not happening with exit, so I changed it to return.

Reply:
@rsevilla87 Also, not sure if this caused any issues. Your last PR test should have failed since it couldn't reach the ES server. Any chance the changes to check_es borked that?